Assignments/Sorting/SelectionSort.playground/Pages/Selection Sort.xcplaygroundpage/Contents.swift

```diff
@@ -9,19 +9,32 @@ Better define an inner function to find the next minimum in the array.
 
 */
 
-func selectionSort(array: [Int]) -> [Int] {
+func selectionSort(var array: [Int]) -> [Int] {
     // You may declare array argument with var keyword so that it is copied.
+    
+    for j in array.startIndex..<array.endIndex - 1 {
```

Comment:
Do we have any particular reason to use `startIndex` and `endIndex` here? `for j in 0..<array.count - 1` would do the same?